transports/http/endpoints/api/merkleroots/endpoints.go

```diff
@@ -25,6 +33,44 @@ func (h *handler) RegisterAPIEndpoints(router *gin.RouterGroup, _ *config.HTTPCo
 	merkle := router.Group("/chain/merkleroot")
 	{
 		merkle.POST("/verify", h.verify)
+		merkle.GET("", h.merkleroots)
+	}
+}
+
+// Merkleroot godoc.
+//
+// @Summary Returns merkleroots for the specified range
+// @Tags merkleroots
+// @Accept */*
+// @Produce json
+// @Success 200 {object} merkleroots.MerkleRootsESKPagedResponse.//TODO: map this in swagger docs
+// @Router /chain/merkleroot [get]
+// @Param batchSize query string false "Batch size of returned merkleroots"
+// @Param lastEvaluatedKey query string false "Last evaluated block height that client has processed"
+// @Security Bearer
+func (h *handler) merkleroots(c *gin.Context) {
+	batchSize := c.DefaultQuery("batchSize", batchSize)
```

Comment:
It would be great if you can add this endpoint as a regression tests "test case".
You can of course add it as a subtask but if you can add it meanwhile in a scope of this one then I would be very happy 😁

Reply:
I'll try to add it in this one

Reply:
added regressions tests. One comment on these. I only included two cases when we fetch from the beginning by not providing evaluationKey and when we give some distant evaluationKey. We could also check for batchSize not being provided but it would result in a very long response string, which I think is irrelevant for these tests. We could also check for the failure paths, i.e merkleroots not found or being stale but I believe testcases implemented in this PR  are enought. WDYT?